### DISCUSSION TITLE
Valtio with urql?

### DISCUSSION THREAD

**Discussion by GollyJer** [pmndrs/valtio#361] [Q&A] [answered]
I'm looking at [Jotai + urql](https://jotai.org/docs/integrations/urql) longingly.

Is there something similar in mind for Valtio? Or does that approach not make sense with proxy state?

**Comment by dai-shi** [comment, marked as answer]
It does make sense, but it's out of the scope of the core library.
Ref: https://github.com/pmndrs/valtio/wiki/How-to-persist-states
I mean it's the scope of 3rd party library. Anyone interested?

**Reply by rpggio** [reply]
Maybe something like https://github.com/pmndrs/suspend-react, but using Valtio proxies for query status?
